samples/boards/nrf52/power_mgr/README.rst

```diff
@@ -0,0 +1,58 @@
+.. _power-mgr-sample:
+
+Power management demo
+#####################
+
+Overview
+********
+
+A sample implementation of a power manager app that uses the Zephyr
+power management infrastructure.
+
+This app will cycle through the various power schemes at every call
+to _sys_soc_suspend() hook function.
+It will cycle through the following states:
+
+1. CPU Low Power State
+   A. CONSTANT LATENCY LOW POWER MODE : This Low Power State triggers CONSTLAT
```

Comment:
Is the word CONSTLAT correct? (or CONSTANT misspelled :)

Reply:
as per nrf52 reference manual (Page 80, 18.3.1 Sub power modes) , they refer it as CONSTLAT task only. So CONSTLAT is correct.